source/feature/openvswitch/native-openvswitch.html.md

```diff
@@ -25,6 +25,8 @@ OVS is to be fully integrated (natively) into oVirt through VDSM, becoming the d
 ## Overview
 The introduction of OVS as an additional network switching solution raises the need to generalize and abstract the requirements from the system networking services and define a spec which considers both options (OVS & native Linux networking).
 The OVS model of arranging networking entities is different from the native Linux networking, but is closer to industry networking modeling.
+Direct connection of VM to the network cannot be done by OvS. In order to connect a VM to an OvS bridge, a user must first define an OvS VM network. 
```

Comment:
"Direct connection of VM to the network cannot be done by OvS"
This is not quite clear.
We can connect a VM to a network (ovirt VM network, represented by a ovs bridge).

Reply:
I have added autodefine feature page to make it more clear hopefully